src/java/picard/sam/markduplicates/EstimateLibraryComplexity.java

```diff
@@ -89,33 +89,27 @@
         programGroup = Metrics.class
 )
 public class EstimateLibraryComplexity extends AbstractOpticalDuplicateFinderCommandLineProgram {
-    static final String USAGE_SUMMARY = "Estimates library complexity.   ";
-    static final String USAGE_DETAILS = "This tool outputs quality metrics for a sequencing library preparation." +
-            "<br /><br />Library complexity refers to the number of unique DNA fragments present in a given library.  " +
-            "Reductions in complexity compromise downstream analyses and can result from either PCR or optical duplicates." +
+    static final String USAGE_SUMMARY = "Estimates library complexity; the numbers of unique molecules in a sequencing library.  ";
+    static final String USAGE_DETAILS = "<p>This tool outputs quality metrics for a sequencing library preparation." +
+            "Library complexity refers to the number of unique DNA fragments present in a given library.  Reductions in complexity resulting " +
+            "from PCR amplification during library preparation, will ultimately compromise downstream analyses " +
+            "via an elevation in the number of duplicate reads.  PCR-associated duplication artifacts can result from: inadequate amounts " +
+            "of starting material (genomic DNA, cDNA, etc.), losses during cleanups, and size selection issues.  " +
+            "Duplicate reads can also arise from optical duplicates resulting from sequencing-machine optical sensor artifacts.</p>  " +
+
+            "<p>This tool attempts to estimate library complexity from sequence of read pairs alone. Does so by sorting all reads by the first N bases (5 by default) of each read and then comparing reads with the first N bases identical to each other for duplicates.  Reads are considered to be duplicates if they match each other with no gaps and an overall mismatch rate less than or equal to MAX_DIFF_RATE (0.03 by default). " +
+            " Reads of poor quality are filtered out so as to provide a more accurate estimate. The filtering removes reads with any no-calls 'N' in the first N bases or with a mean base quality lower than MIN_MEAN_QUALITY across either the first or second read.  Unpaired reads are ignored in this computation. " +
```

Comment:
"no-calls 'N' in the first N bases" -> may want to disambiguate that these Ns are unrelated